apps/namadillo/src/App/Ibc/IbcTransfer.tsx

```diff
@@ -84,15 +87,17 @@ export const IbcTransfer = (): JSX.Element => {
     selectedAssetAddress ? userAssets?.[selectedAssetAddress] : undefined;
 
   const availableAssets = useMemo(() => {
-    if (!userAssets) return undefined;
+    if (!enabledAssets || !userAssets) return undefined;
+
     const output: AddressWithAssetAndAmountMap = {};
     for (const key in userAssets) {
-      if (registry?.assets.assets.find((a) => a.base === key)?.base) {
```

Comment:
https://github.com/anoma/namada-interface/commit/a484b1d23a249fd0273432ecb23f552ff6985c25#diff-575a5c2575fb2f1e0bc8cfed3eeb1c570eead4d3c9f1b574e8a4997c7df3e647

restoring @mateuszjasiuk old way of doing it